src/app/shared/media/component/media.component.ts

```diff
@@ -1,18 +1,39 @@
-import { Component, Input } from '@angular/core';
+import {
+  AfterViewInit,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  Input,
+  OnDestroy,
+  Renderer2,
+  ViewChild,
+} from '@angular/core';
 import { DomSanitizer, SafeUrl } from '@angular/platform-browser';
-import { BehaviorSubject, ReplaySubject } from 'rxjs';
-import { map } from 'rxjs/operators';
+import { Capacitor } from '@capacitor/core';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { PreviewVideo } from '@numbersprotocol/preview-video';
+import { BehaviorSubject, EMPTY, ReplaySubject, combineLatest, of } from 'rxjs';
+import { catchError, filter, map, switchMap, tap } from 'rxjs/operators';
 import { MimeType } from '../../../utils/mime-type';
 
+@UntilDestroy()
+@UntilDestroy()
```

Comment:
There is 2 `@UntilDestroy()`

Reply:
@shc261392, I will remove one and re-commit

Reply:
@shc261392 , done [chore(media.component): remove 2nd @UntilDestroy](https://github.com/numbersprotocol/capture-lite/pull/2868/commits/9bd949f1ad0007f546dbdb069a8b3d284cfd3a2b)